docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,223 @@
+# Natural Language Processing (Nlp)
```

Comment:
Global Question : Why "Nlp", not "NLP"?
Is there a rule to write in this way?

Reply:
no , we just use the namespace(Tizen.Nlp)  in code

Reply:
I see. As this page is a guide for the "Natural Language Processing", how about changing "Nlp" to "NLP" here?
I searched "Nlp" in Google, it looks like "NLP" is commonly used.